.github/workflows/generate.yml

```diff
@@ -0,0 +1,55 @@
+name: Regenerate Asset Data and PR
+
+on:
+  workflow_dispatch:
```

Comment:
This one can be kept if we want to, allows for the action to be ran manually (either via GH's UI or via the GH cli)